index.js

```diff
@@ -161,30 +161,23 @@ async function init() {
 			return;
 		}
 
-		if (type !== 'tree' || !ref) {
+		if (type === 'blob') {
```

Comment:
There are many variations. Again the type must be specifically tree or nothing, the old condition was correct.